apps/backend-sls/prisma/seed.ts

```diff
@@ -37,30 +38,34 @@ async function main(prisma: Prisma.PrismaClient) {
     update: {},
   })
 
-  const participant = await prisma.participant.upsert({
-    where: {
-      id: '6f45065c-667f-4259-818c-c6f6b477eb48',
-    },
-    create: {
-      id: '6f45065c-667f-4259-818c-c6f6b477eb48',
-      email: 'testuser@bf.uzh.ch',
-      password: hash2,
-      username: 'rschlaefli',
-      avatar: 'apple-icon-180.png',
-      participations: {
+  const participants = await Promise.all(
+    PARTICIPANT_IDS.map((id, ix) => {
```

Comment:
creates 10 testusers, all with testuser0 to testuser10 and same password (testing)